enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,269 @@
+inbound access and this is not configurable.
+
+Basic Walkthrough:
+1. Create a paid IBM Cloud Account (PowerVS - or any infrastructure service, is not available for free accounts).
+2. Create Power VS Service instance in specific region
+3. Create private network in the service instance
+4. [Upload/Create<sup>*</sup>](#coreos-ova-considerations) the required boot images.
+5. Create VM by specifying configurable values:
+    - \# of CPU / Cores
+    - Shared vs. Dedicated CPUs
+    - Server class (s922 vs. e980)
+    - Machine type
+    - \# of Memory
+    - Boot Image
+    - Additional disk volume(s)
+    - Storage Type (Tier 1 or 3)
+    - Network (public, private)
+
+*Note*: Disk size of boot image == size of boot image
+
+[Power VS Reference Doc][power-vs-reference-doc]
+
+### Open Questions
+
+#### OVA Import Time
+Is 5 minutes for important a target deployment image satisfactory for an IPI deployment. Additonally, is it even
+possible for the Power VS team to reduce the current import time from 30 minutes to 5 (for the purposes of meeting this
+requirement)?
+
+#### Deployment Time
+What is the maximum allowable time for a successful deployment to take. Is under an hour sufficient?
+
+#### DHCP in the IBM Cloud
+How soon will we have DHCP in the Power VS platform. If we can't get DHCP, do we have other options?
+
+#### Flavors or Granular Control
+One thing that is being discussed is whether it would be better to expose a subset of cloud instance "flavors" with
+pre-defined compute, storage, CPU cores, etc., or to have the installer mirror the current instance-creation requirement
+for Power VS.
+
+One thing that is clear is that limiting the number of installation options is key to long term sustainability.
+
+### Test Plan
+
+Multi-Arch currently tests results by running upgrade and e2e tests with the OpenShift Prow CI environment. Our test
+plan would extend this to include these IPI scenarios.
+
+### Graduation Criteria
+
+- Our initial target is to work towards a `Dev Preview`, then graduate towards future milestones. This is expected
+to take a total of three releases, but could be more or less depending on how quickly we meet the requirements
+enumerated in the graduation criteria.
+- `Dev Preview` will graduate to  `Tech Preview` and `Tech Preview` to `GA` in OpenShift
+- [Deprecation policy][deprecation-policy]
+
+[deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
+
+#### Dev Preview Requirements
+- [ ] Implemented APIs necessary for IPI installation
+- [ ] DHCP requirement solved by the Power VS team
+- [ ] Builds of CoreOS are available in the IBM Cloud for installation
+- [ ] First pass at defining which installation criteria belong in the installer and which are day 2 configurations
+- [ ] Developer documentation
+- [ ] Plan defined to get feedback from developers
+- [ ] Sufficient time for feedback (6 weeks)
```

Comment:
The OpenShift team raised to us in the 4.2 days that it takes a month for CI to "soak" / stabilize. Since our sprints take 3 weeks, I bumped this to align with sprint boundaries. I also felt that 6 weeks was a better target for internal / external customer feedback gathering.

would like to hear other's thoughts on this.

Reply:
I've made some tweaks to this when reformatting onto a Tech + GA format, rather than Dev->Tech->GA

The key with the 6 weeks timer is that it starts when the deliverable is shipped, and the intent is to ensure we have enough time to gather feedback from any prototypes / early adopters and that we're acknowledging and acting upon that feedback.

I can't remember which enhancements proposal I based this on, but I had seen a precedent set for this. That said, I think it might be fair to relax this requirement if there is another way to capture the essence of that commitment to driving quality through the feedback we get. At least insofar as CI, my understanding is that we've found that setting time boundaries ensure we have enough data to actually make determinations about the quality of the release. IDK if that is true for tech preview feedback, but I imagine if we dropped the Tech Preview late in a release cycle, we would want at least some time to absorb feedback before slapping a GA label on.

It's worth noting that platform deliverables have not always been synchronous with GA of an OpenShift release. For example, the ARM developer preview is out now, and that is based on 4.9.